dashboard/app/controllers/datablock_storage_controller.rb

```diff
@@ -155,10 +154,8 @@ def add_column
     column_name = params[:column_name]
 
     table = DatablockStorageTable.find([params[:channel_id], params[:table_name]])
-    columns = JSON.parse(table.columns)
-    unless columns.include? column_name
-      columns << column_name
-      table.columns = columns.to_json
+    unless table.columns.include? column_name
+      table.columns << column_name
```

Comment:
At some point we screwed up and started saving a serialized-JSON string to table.columns, instead of saving the straight JSON. The result was that columns was in our DB as a JSON string like `'["id","name"]'`, not as a JSON object like `["id","name"]`.

Because this PR tries to "get columns right", I've unwound all those mistakes, and now table.columns is an object, not a string of a JSON-ified object.

Sorry for the extra noise in the PR tho lol